enterprise/templates/enterprise/grant_data_sharing_permissions.html

```diff
@@ -31,7 +31,11 @@
                         </h2>
                     </div>
                     <span class="text">
-                        ${_("In order to link your account, we need consent to share your course data with {provider}.").format(provider=sso_provider)}
+                        % if not course_id:
+                            ${_("In order to link your account, we need consent to share your course data with {provider}.").format(provider=ec_name)}
+                        % else:
+                            ${_("In order to access {course_name}, we need consent to share your course data with {provider}.").format(provider=ec_name, course_name=course_name)}
```

Comment:
@griffresch see this comment regarding the term "manual enrollment" ^^  (no issue/worries @haikuginger, just a point I'm trying to make)